justfile

```diff
@@ -72,6 +72,23 @@ benchmark-pallet chain="polimec-paseo-local"  pallet="pallet-dispenser":
       --output=pallets/{{ replace(pallet, "pallet-", "") }}/src/weights.rs \
       --template=./.maintain/frame-weight-template.hbs
 
+
+benchmark-extrinsics pallet="pallet-funding" extrinsics="*" :
+    cargo run --features runtime-benchmarks --profile=production -p polimec-node benchmark pallet \
+      --chain=polimec-paseo-local \
```

Comment:
What is the reasoning behind choosing to use `polimec-paseo-local`?

Reply:
I remember having problems using the original polimec runtime because that one was made for an older runtime.